src/main/java/com/crud/tasks/service/MailCreatorService.java

```diff
@@ -17,20 +20,45 @@
     @Autowired
     @Qualifier("templateEngine")
     private TemplateEngine templateEngine;
-    private Context context = new Context();
+    private Context contextTrello = new Context();
+
+    private List<String> functionality = new ArrayList<>();
 
     public String buildTrelloCardEmail(String message) {
-        context.setVariable("message", message);
-        context.setVariable("tasks_url", "http://localhost:5000/");
-        context.setVariable("button", "Visit website");
-        context.setVariable("goodbye", "Best regards: " + adminConfig.getAdminName());
-        context.setVariable("details", "Company Details:");
-        context.setVariable("admin_name", adminConfig.getAdminName());
-        context.setVariable("app_name", "Company name: " + companyDetails.getAppName());
-        context.setVariable("app_description", "Company description: " + companyDetails.getAppDescription());
-        context.setVariable("company_mail", "Company e-mail: " + companyDetails.getMail());
-        context.setVariable("company_mobile", "Company mobile: " + companyDetails.getMobile());
-        context.setVariable("show_button", false);
-        return templateEngine.process("mail/created-trello-card-mail", context);
+        contextTrello.setVariable("message", message);
+        makeListForIteration();
```

Comment:
functionality jest zmienna klasowa, ktora 'puchnie' przy kazdym wywolaniu metody buildTrelloCardEmail. Jesli wartosci jest zawsze ta sama, to powinna byc stala inicjowana w konstruktorze